influxdb/influxdb08/dataframe_client.py

```diff
@@ -13,6 +13,16 @@
 from .client import InfluxDBClient
 
 
+def total_seconds(td):
```

Comment:
Hi, 
thank you for your time.
I tried to add `total_seconds` method to `InfluxDBClient` class, t will be more elegant solution, but problem is that `InfluxDBClient` class is not imported in **line_protocol.py**, and can't be because of "circular import error". 
So the `total_seconds` function must be separately specified in **line_protocol.py**.

See in new commit.